src/integration/tests/firstTest.test.tsx

```diff
@@ -13,39 +12,114 @@ import {
+    function formatDate(date = new Date()) {
+      const options: Intl.DateTimeFormatOptions = {
+        month: "long",
+        year: "numeric",
+      };
+      const formattedDate = date.toLocaleString("pt-BR", options);
+      const [month, year] = formattedDate.split(" de ");
+      return `${month.charAt(0).toUpperCase()}${month.slice(1)} de ${year}`;
+    }
+
+    const dateFormat = infoArea.getByText(formatDate());
+    expect(dateFormat).toBeInTheDocument();
+  });
+
+  it("should be possible to view expense values in green in the Balanço", () => {
+    const infoArea = within(screen.getByTestId(INFO_AREA_ID));
+    const infoBalance = infoArea.getByTestId(BALANCE_VALUE_ITEM_ID);
+    expect(infoBalance).toHaveStyle("color: green");
+  });
+
+  it("should be possible to view expense values in black in the Receitas", () => {
+    const infoArea = within(screen.getByTestId(INFO_AREA_ID));
+    const revenueValue = infoArea.getByTestId(REVENUE_VALUE_ITEM_ID);
+    expect(revenueValue).toHaveStyle("color: #000");
+  });
+
+  it("should be possible to view expense values in black in the Despesas", () => {
+    const infoArea = within(screen.getByTestId(INFO_AREA_ID));
+    const expenseValue = infoArea.getByTestId(EXPENSE_VALUE_ITEM_ID);
+    expect(expenseValue).toHaveStyle("color: #000");
+  });
+
+  it("should be possible to view the fields Data, Categoria, Titulo, Valor, and the Adicionar button.", () => {
+    const inputArea = within(screen.getByTestId(INPUT_AREA_ID));
+
+    const inputData = inputArea.getByText(DATE_LABEL);
+    const inputCateg = inputArea.getByText(CATEGORY_LABEL);
+    const inputTitutlo = inputArea.getByText(TITLE_LABEL);
+    const inputValor = inputArea.getByText(VALUE_LABEL);
+    const inputAdd = inputArea.getByText(ADD_LABEL);
+
+    expect(inputData).toBeInTheDocument();
+    expect(inputCateg).toBeInTheDocument();
+    expect(inputTitutlo).toBeInTheDocument();
+    expect(inputValor).toBeInTheDocument();
+    expect(inputAdd).toBeInTheDocument();
+  });
+
+  it("should be possible to view the report with columns for Data, Categoria, Titulo, and Valor.", () => {
+    const tableArea = within(screen.getByTestId(TABLE_AREA_ID));
 
+    const tableData = tableArea.getByText(DATE_LABEL);
+    const tableCateg = tableArea.getByText(CATEGORY_LABEL);
+    const tableTitle = tableArea.getByText(TITLE_LABEL);
+    const tableValor = tableArea.getByText(VALUE_LABEL);
 
-  beforeAll(() => {
-    const originalAlert = window.alert;
-    window.alert = jest.fn();
-    afterAll(() => {
-      window.alert = originalAlert;
-    });
+    expect(tableData).toBeInTheDocument();
+    expect(tableCateg).toBeInTheDocument();
+    expect(tableTitle).toBeInTheDocument();
+    expect(tableValor).toBeInTheDocument();
   });
-  it("should be able insert the salario", () => {
+
+  it("should be able to insert an revenue item.", () => {
     const now = new Date(Date.now());
```

Comment:
You could create your own function to format "now" date and return day, month and year